configure

```diff
@@ -21,6 +21,15 @@ check_opts()
     echo "MAX_DISPATCHER_ACTIONS:=${MAX_DISPATCHER_ACTIONS}" >>$CONFIG
 }
 
+check_bpf_target()
+{
+    if [ -z $BPF_TARGET ]; then
+        BPF_TARGET=bpf
+    fi
+    echo "BPF_TARGET:=${BPF_TARGET}" >>$CONFIG
+    echo "$BPF_TARGET"
+}
```

Comment:
Instead of a whole new function, let's just put this into check_opts and add a `BPF_TARGET?=bpf` to defines.mk.

Reply:
updated

Reply:
wait something is wrong

Reply:
`BPF_TARGET?=bpf` Does not work?

Reply:
You're setting it to empty in the configure script. You want '-n' instead of '-z'

Reply:
I think I have to completely remove it? Then it is working.

Reply:
Hmm, no I think we do want this setable in configure so it persists. But we can put a default in there to avoid setting it to an empty variable. So in check_opts() you can add a default like of the other variables: `: ${BPF_TARGET:=bpf}` 

and then echo it to config.mk. And then move the `BPF_TARGET?=bpf` to the top of defines.mk (along with CFLAGS, before the includes) and I think things should work in all cases...

Reply:
Changed. :)